test/common_distributed.py

```diff
@@ -110,6 +111,49 @@ def get_timeout(test_id):
     return TIMEOUT_OVERRIDE.get(test_id.split('.')[-1], TIMEOUT_DEFAULT)
 
 
+def simple_sparse_reduce_tests(rank, world_size, num_inputs=1):
```

Comment:
Didn't look into this method, this is just moving the function and there is no changes here, right?

Reply:
yes, just moved the function